src/main/java/frc/robot/RobotContainer.java

```diff
@@ -138,7 +139,9 @@ private void configureButtonBindings() {
     if (Robot.armEnabled) {
       OI.Driver.getArmHighButton().onTrue(new ProxyCommand(() -> armSubsystem.handleHighButton()));
       OI.Driver.getArmMidButton().onTrue(new ProxyCommand(() -> armSubsystem.handleMidButton()));
-      OI.Driver.getArmIntakeButton().onTrue(new ProxyCommand(() -> armSubsystem.handleIntakeButton()));
+      OI.Driver.getArmConeIntakeButton().onTrue(new ProxyCommand(() -> armSubsystem.handleConeIntakeButton()).andThen(() -> intakeSubsystem.setMode(true)));
+      OI.Driver.getArmCubeIntakeButton().onTrue(new ProxyCommand(() -> armSubsystem.handleCubeIntakeButton()).andThen(() -> intakeSubsystem.setMode(false)));
```

Comment:
Needs intake subsystem requirement and `if (Robot.intakeEnabled)` check

Reply:
Also, since these commands use different subsystems and don't really depend on eachother, may as well make them a ParallelCommandGroup (alongWith instead of andThen)